widgets/accordion/js/accordion.js

```diff
@@ -142,7 +142,12 @@ jQuery( function ( $ ) {
 						var panel = panels[ i ];
 						var anchor = $( panel ).data( 'anchor' );
 						var anchors = window.location.hash.substring(1).split( ',' ); 
-						if ( anchor && $.inArray( anchor.toString(), anchors ) > -1 ) {
+						
+						var anchor = anchors.filter( function ( index, element ) {
+							return decodeURI( anchor.toString() ) === decodeURI( index );
```

Comment:
This is decoding the integer index of the element and comparing it to the decoded `anchor` string . It should probably be something more like `return decodeURI( anchor.toString() ) === decodeURI( $( element ).data( 'anchor' ) );`.